java/src/main/java/scenarios/crewcreation/CharacterCreation.java

```diff
@@ -148,7 +148,7 @@ private static void printAbilityScoreFlavorText(int abilityScoreValue, String ab
         System.out.println();
     }
 
-    private static void printFlavorPhysicalTextScore (int abilityScoreValue){
+    private static void printPhysicalScoreFlavorText (int abilityScoreValue) {
```

Comment:
I'll change this for you when I add the unit tests, but I just realized that there is also a space in between the method name and the parentheses that shouldn't be here.

Reply:
I just noticed, you had done this on a few other methods too! Just as a reminder, there shouldn't be any space between the method name and the parentheses(although this is just a stylistic thing and won't have an effect on your code, it is ubiquitous).